zebra/main.c

```diff
@@ -179,6 +180,13 @@ static void sigint(void)
 
 	access_list_reset();
 	prefix_list_reset();
+	/*
+	 * Before routemap finish, call zebra_rmap_upd_disable_and_cleanup.
+	 * This will
+	 * 1 set rmap upd timer to 0 so that rmap update wont be scheduled again
+	 * 2 Put off the rmap update thread
+	 */
+	zebra_rmap_upd_disable_and_cleanup();
```

Comment:
let's call the function zebra_routemap_finish() to follow naming structure used in FRR.  route_map_finish should be in this function too.